packages/esm-appointments-app/src/home-appointments/appointments-list.scss

```diff
@@ -342,4 +344,8 @@ html[dir='rtl'] {
       }
     }
   }
+
+  .seeAllLink {
+    left: -22rem;
```

Comment:
Can we align it respect to the parent instead of the page, otherwise this might break again in the future or even in the LtR format?

Reply:
Probably not too bad for now.